src/main/java/com/minecolonies/coremod/util/LanguageHandler.java

```diff
@@ -55,33 +89,8 @@ public static void sendPlayerMessage(@NotNull final EntityPlayer player, final S
      */
     public static String format(final String key, final Object... args)
     {
-        return String.format(getString(key), args);
-    }
-
-    /**
-     * Localize a non-formatted string.
-     *
-     * @param key unlocalized key.
-     * @return Localized string.
-     */
-    public static String getString(final String key)
-    {
-        return getString(key, key);
-    }
-
-    /**
-     * Localize a non-formatted string.
-     *
-     * @param key          unlocalized key.
-     * @param defaultValue the value to return if no key is found.
-     * @return Localized string.
-     */
-    @SuppressWarnings("deprecation")
-    public static String getString(final String key, final String defaultValue)
-    {
-        //todo: use TextComponentTranslation like mojang wants us to
-        //using fully qualified name to remove deprecation warning on import
-        return net.minecraft.util.text.translation.I18n.translateToLocal(key);
+        String result = new TextComponentTranslation(key, args).getFormattedText();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'result' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)